RecoTracker/TrackProducer/test/BuildFile.xml

```diff
@@ -1,6 +1,9 @@
 <use name="TrackingTools/Records"/>
+<use name="RecoTracker/Record"/>
 <use name="FWCore/Utilities"/>
 <use name="DataFormats/TrackReco"/>
+<use name="RecoTracker/TrackProducer"/>
+<use name="TrackingTools/TransientTrack"/>
```

Comment:
are changes in this file really a part of this PR?

Reply:
I dont think so, I can rebase later